README.md

```diff
@@ -56,7 +56,7 @@ When learning CS there are some useful sites you must know to get always informe
    * [CS 106B Coding Style Guide](http://stanford.edu/class/archive/cs/cs106b/cs106b.1158/styleguide.shtml) : must see for those who create spaghetti
    * [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html)
    * [What are some bad coding habits you would recommend a beginner avoid getting into?](https://www.reddit.com/r/learnprogramming/comments/1i4ds4/what_are_some_bad_coding_habits_you_would/)
-   * [Good C programming habits. • /r/C_Programming](https://www.reddit.com/r/C_Programming/comments/1vuubw/good_c_programming_habits/)
```

Comment:
Did you intend to remove the "C_" portion of the r/C_Programming subreddit link? 
Since the link is not from the r/programming subreddit but rather the /r/C_Programming subreddit I believe that should be corrected.